constrain-lib/src/main/java/org/gradle/dependency/constrain/lib/model/LoadedConstraint.java

```diff
@@ -18,14 +18,22 @@
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+import java.util.function.BiPredicate;
 
 import static java.util.Objects.requireNonNull;
 
 /**
  * Represents a constraint loaded from the constraints.xml file.
  */
 public final class LoadedConstraint {
```

Comment:
Because I don't want to name collide with the Gradle `DependencyConstraint` type. I'm open to any other ideas here on naming. I'm not particularly a fan of this myself, but it seemed like a good way to disambiguate the two.

Reply:
`SecurityDependencyConstraint` or something along those lines.

Reply:
I see this being used outside of the scope of just security in the future. So I don't think that making the name specifically security related is the right move here. Any other naming suggestions?

Reply:
Is it a `GlobalDependencyConstraint`? If you can't find a more fitting name, I would still go with `DependencyConstraint` as it lives in a different package and would match the name of the json schema.